libr/debug/p/debug_windbg.c

```diff
@@ -117,29 +112,29 @@ static int r_debug_windbg_attach (RDebug *dbg, int pid) {
 	}
 
 	// Handshake
-	if (!windbg_sync(wctx)) {
-		eprintf("Could not connect to windbg\n");
-		windbg_ctx_free(wctx);
+	if (!windbg_sync (wctx)) {
+		eprintf ("Could not connect to windbg\n");
+		windbg_ctx_free (wctx);
 		return false;
 	}
-
-	if (!windbg_read_ver(wctx)) {
-		windbg_ctx_free(wctx);
+	if (!windbg_read_ver (wctx)) {
+		windbg_ctx_free (wctx);
 		return false;
 	}
+
 	// Make r_debug_is_dead happy
 	dbg->pid = 0;
 	return true;
 }
 
-static int r_debug_windbg_detach (RDebug *dbg, int pid) {
+static int r_debug_windbg_detach(RDebug *dbg, int pid) {
+	eprintf ("Detaching...\n");
 	return true;
 }
 
 static char *r_debug_windbg_reg_profile(RDebug *dbg) {
 	if (!dbg) return NULL;
-	if (dbg->arch && strcmp (dbg->arch, "x86"))
-		return NULL;
+	if (dbg->arch && strcmp (dbg->arch, "x86")) return NULL;
 	if (dbg->bits == R_SYS_BITS_32) {
 #include "native/reg/windows-x86.h"
```

Comment:
yes, its an include directive from the c preprocessor. thats common and standard